### ISSUE TITLE
No interaction event from item display furnitures

### ISSUE THREAD

**Issue by Xiao-MoMi** [PluginBugs/Issues-ItemsAdder#3990]
### Terms

- [X] I'm using the very latest version of ItemsAdder and its dependencies.
- [X] I am sure this is a bug and it is not caused by a misconfiguration or by another plugin.
- [X] I've looked for already existing issues on the [Issue Tracker](https://github.com/PluginBugs/Issues-ItemsAdder/issues) and haven't found any.
- [X] I already searched on the [plugin wiki](https://itemsadder.devs.beer/) to know if a solution is already known.
- [X] I already searched on the [forums](https://forum.devs.beer/) to check if anyone already has a solution for this.
- [X] I tested this with just ItemsAdder and its dependencies and with a vanilla client of the same version as the Server.

### What happened?

No furniture interaction events with item display are triggered

### Steps to reproduce the issue

I think you may have forgot to handle raytracing for interactions

### Server version

This server is running Paper version 1.21.1-52-master@e08e667 (2024-08-26T18:02:06Z) (Implementing API version 1.21.1-R0.1-SNAPSHOT)

### ItemsAdder Version

ItemsAdder version 4.0.2-beta-release-10

### ProtocolLib Version

ProtocolLib version 5.3.0-SNAPSHOT-726

### LoneLibs Version

LoneLibs version 1.0.58

### Full Server Log

.

**Comment by Andre601** [comment]
Please share your `latest.log` through https://mclo.gs as it help us getting all the necessary info required to solving this issue.

**Comment by LoneDev6** [comment]
Can you add more details to explain what you mean?
When did you notice that no interaction is triggered?
Are you using Spigot API? Configurations?
Please explain, thanks

**Comment by Xiao-MoMi** [comment]
No, I knew that display entities can't be interacted and Spigot API won't have interact events related to display entities. I am saying about the FurnitureInteractEvent.

**Comment by LoneDev6** [comment]
So `FurnitureInteractEvent` is not triggered at all?
Can you provide an example yml of a furniture that has this issue?
